src/main/java/com/notably/logic/suggestion/SuggestionEngineImpl.java

```diff
@@ -61,56 +55,54 @@ public void suggest(String userInput) {
      * @param userInput The user's input.
      * @return The corresponding SuggestionCommand.
      */
-    private SuggestionCommand parseCommand(String userInput) {
+    private Optional<SuggestionCommand> parseCommand(String userInput) {
         final Matcher matcher = BASIC_COMMAND_FORMAT.matcher(userInput.trim());
         if (!matcher.matches()) {
-            return new ErrorSuggestionCommand(
-                    "Invalid command format. To see the list of available commands, type: help");
+            model.setResponseText("\"" + userInput + "\" is an invalid command format. "
+                    + "To see the list of available commands, type: help");
```

Comment:
Should we make this message a static constant string? Looks like you use it multiple places here